### PR TITLE
Mcp 2273

### PR DESCRIPTION
**Pull request by wlin2019**
Fix BIP test endpoints in swagger UI.

## What was the problem?

Three BIP test endpoints (create a contention, update a contention, and set claim status to RFD) return 500 error.

Associated tickets or Slack threads:

MCP-2273

## How does this fix it?[^1]
Those endpoints used to work. They were broken accidentally by a code style fix in mid December. Revert the accidental change to fix the problem. In addition, I also modify the mock service to return some message to indicate the result from mock service.

## How to test this PR
- Step 1. Build and launch the application as usual.
- Step 2. Test creat-contetion, update-contention, setRFD endpoints. It should not return 500 when proper data is given.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by ianwolson — APPROVED** [review]

File: shared/domain/src/main/java/gov/va/vro/model/bip/BipFileUploadResp.java [hunk 1]
```diff
@@ -11,6 +12,7 @@
  * @author warren @Date 12/7/22
  */
 @RequiredArgsConstructor
+@AllArgsConstructor
```
Comment (dimitri-amida) [inline shared/domain/src/main/java/gov/va/vro/model/bip/BipFileUploadResp.java:15]: The constructor annotations are not needed

**Review by dimitri-amida — APPROVED** [review]
Minor comments. These changes do not break anything.

File: controller/src/main/java/gov/va/vro/controller/BipController.java [hunk 2]
```diff
@@ -116,7 +116,7 @@ public ResponseEntity<BipClaimContentionsResponse> getContentions(@Valid String
   public ResponseEntity<BipContentionUpdateResponse> updateContentions(
       @Valid BipUpdateClaimContentionPayload payload) {
     log.info("update a contention for claim ID {}", payload.getClaimId());
-    log.info("contention to update:\n {}", payload.getContention());
+    //    log.info("contention to update:\n {}", payload.getContention());
```
[the remark this answers is not in the document]
Comment (wlin2019) [reply controller/src/main/java/gov/va/vro/controller/BipController.java]: My bad. Thanks for catching it. Removed it.

File: controller/src/main/java/gov/va/vro/controller/BipController.java [hunk 3]
```diff
@@ -143,7 +143,7 @@ public ResponseEntity<BipContentionUpdateResponse> updateContentions(
   public ResponseEntity<BipContentionCreationResponse> createContentions(
       @Valid BipCreateClaimContentionPayload payload) {
     log.info("Create a contention for claim ID {}", payload.getClaimId());
-    log.info("contention to create: \n {}", payload.getContention());
+    //    log.info("contention to create: \n {}", payload.getContention());
```
[the remark this answers is not in the document]
Comment (wlin2019) [reply controller/src/main/java/gov/va/vro/controller/BipController.java]: done

File: shared/domain/src/main/java/gov/va/vro/model/bip/BipFileUploadResp.java (hunk 1 above)
Comment (wlin2019) [reply shared/domain/src/main/java/gov/va/vro/model/bip/BipFileUploadResp.java:15]: AllargsConstructor is used. Without it, error shows up. Maybe @RequiredArgsContructor is not needed.

**Review by alan-amida — APPROVED** [review]
Looks good to me 👍

**Pull request merged**